.github/CODEOWNERS

```diff
@@ -28,14 +28,17 @@ Objects/type*                 @markshannon
 Objects/codeobject.c          @markshannon
 Objects/frameobject.c         @markshannon
 Objects/call.c                @markshannon
-Python/ceval.c                @markshannon
+Python/ceval*.[ch]            @markshannon @gvanrossum
```

Comment:
GitHub now says "This CODEOWNERS file contains errors" because of this line

Reply:
Whoops, sorry. Do you know what’s wrong? I am AFK.

Reply:
I'll do a Google :-)

Reply:
Looks like it's [this](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax):

> Warning: There are some syntax rules for gitignore files that *do not work* in CODEOWNERS files:
> 
> - Escaping a pattern starting with `#` using `\` so it is treated as a pattern and not a comment
> - Using `!` to negate a pattern
> - Using `[ ]` to define a character range

Reply:
I'll make a PR to fix it

Reply:
I filed #112210